default.nix

```diff
@@ -139,7 +138,7 @@ let
 
       echo "build-users-group = " > \$dir/conf/nix.conf
       echo "experimental-features = nix-command flakes" >> \$dir/conf/nix.conf
-      echo "ignored-acls = security.selinux system.nfs4_acl" >> \$dir/conf/nix.conf
+      echo "ignored-acls = security.selinux system.nfs4_acl security.csm" >> \$dir/conf/nix.conf
```

Comment:
Is this setting really needed? reading the nix source code this list seems to be blocked by default already.